### PR TITLE
Bug 1170658 - title bar test

### PR DESCRIPTION
**Pull request by hfiguiere**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3f80fdcc561ddaa0544c54a9581768b6fb509bbf

**Comment by hfiguiere** [comment]
On top of the other PR pending review as it depends on it.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e65d91db5ccbaaac5c8d8d6e74a03bcfa6c05185

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=86e2c10c23344b9322f4921b52fdf2a7da96dbfb

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=069033ca794de5ac00c5a712eb48a9190f97996e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=269e11a0365cf2b607421358c2ec5294a7299ac5

File: apps/music/test/unit/metadata/mock_album_art_cache.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (jimporter) [inline apps/music/test/unit/metadata/mock_album_art_cache.js]: Nit: There's an empty line at the top here.

File: apps/music/test/unit/metadata/mock_album_art_cache.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+/* exported MockAlbumArtCache */
+'use strict';
+
+
+var MockAlbumArtCache = {
+  getThumbnailBlob: function() {
+    // Mock: return a fake promise.
+    return {
```
Comment (jimporter) [inline apps/music/test/unit/metadata/mock_album_art_cache.js]: As in the other PR, return a real Promise here with `return Promise.resolve();`

Comment (hfiguiere) [inline apps/music/test/unit/metadata/mock_album_art_cache.js]: Yeah I'll make sure both have since one depend on the other.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7f34b4e26343ac15073b5c08cc0c209e09dca910

File: apps/music/test/unit/metadata/mock_album_art_cache.js (hunk 1 above)
Comment (hfiguiere) [inline apps/music/test/unit/metadata/mock_album_art_cache.js]: it's gone now

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9401669d6f2405e99454cf5fff73a6f90d0309a2

**Pull request merged**